src/data/finset/pointwise.lean

```diff
@@ -464,12 +498,24 @@ s.zero_mul_subset.antisymm $ by simpa [mem_mul] using hs
 end mul_zero_class
 
 section group
-variables [group α] {s t : finset α} {a b : α}
+variables [group α] [division_monoid β] [monoid_hom_class F α β] (m : F) {s t : finset α} {a b : α}
 
 /-! Note that `finset` is not a `group` because `s / s ≠ 1` in general. -/
 
-section decidable_eq
-variables [decidable_eq α]
+@[simp, to_additive] lemma one_mem_div_iff : (1 : α) ∈ s / t ↔ ¬ disjoint s t :=
+by rw [←mem_coe, disjoint_iff_disjoint_coe, coe_div, set.one_mem_div_iff]
+
+@[to_additive] lemma not_one_mem_div_iff : (1 : α) ∉ s / t ↔ disjoint s t :=
+one_mem_div_iff.not_left
+
+@[to_additive] lemma nonempty.one_mem_div (h : s.nonempty) : (1 : α) ∈ s / s :=
```

Comment:
Is the `subset` statement worth having too?

Reply:
What do you have in mind? This is already the `subset` statement (because `s / s` is not `1` in general).

Reply:
`1 ⊆ s / s`

Reply:
Not worth it IMO. `1 ⊆ s` is a rare condition to want and  we have `finset.one_subset` to convert between both. But yeah that makes the statement look more like the `filter` one.

Reply:
Fair enough